### DISCUSSION TITLE
Unable to add backslashed Material Icon code string to content with arbitrary value

### DISCUSSION THREAD

**Discussion by gyohza** [tailwindlabs/tailwindcss#9867] [Help]
I'm trying to set up a Material Icon as the `:before` pseudo-element in an Angular component.

I need to set the content as `\e5cf`, as it's the character code for the `expand_more` icon from the [Material Icons font family](https://fonts.google.com/icons?icon.query=expand_more). I don't have access to the component itself, so I'm working it like this:

```html
<div
  class="
    [&_.inner-class]:before:font-['Material_Icons']
    [&_.inner-class]:before:content-['\e5cf']
  "
>
  <app-component>
    A man painted silver who makes robot noises!
  </app-component>
</div>
```

The problem is the whole string is just wiped clean (i.e. `--tw-content: '';`) since backslash is the escape character - so I figured I could just escape it like `\\e5cf` and it would be fine, but nope - now the content is just a single `\`, and the rest of the string is still wiped clean.

I was able to work around this by begrudgingly adding `--expand-icon: '\e5cf';` to my SCSS file and using `content-[var(--expand-icon)]` instead.

To me, that looks pretty much like a bug, but is there a more elegant way I can work this right now without actually adding things to my config file or globals?

---

Bonus note: the `content-` docs should state (and very clearly) that the order in which you add the classes matter, as setting the content first thing will have the next classes override the content with `''`. This was also a source of frustration for me for a few minutes until I realized what was going on.

**Comment by wongjn** [comment]
Backslashes should work fine, I've just tried it in a Tailwind Play sandbox and locally via Tailwind CLI with the exact same class you supplied in the initial post, and they render out fine. This leads me to believe that it could be something after Tailwind CSS in your build pipeline that is somehow interfering with the output.
